bedrock/exp/templates/exp/firefox/index.de.html

```diff
@@ -51,25 +51,25 @@ <h2 class="c-showcase-title">Das sind die Produkte der Firefox-Familie</h2>
         <ul class="c-showcase-list">
           <li>
             <a class="mzp-c-cta-link" href="{{ url('firefox.new') }}" data-cta-type="link" data-cta-text="Browsers">
-              <img alt="" src="{{ static('protocol/img/logos/firefox/browser/logo-md.png') }}" height="40" width="40"><br>
+              <img alt="" src="{{ static('node_modules/@mozilla-protocol/core/protocol/img/logos/firefox/browser/logo-md.png') }}" height="40" width="40"><br>
```

Comment:
If we copied over the Protocol images to `/protocol/` like we do currently, would this mean we don't have to touch so many files? Or is this just generally tidier to do long term?